node/chainSimulator/chainSimulator.go

```diff
@@ -176,6 +176,9 @@ func (s *simulator) incrementRoundOnAllValidators() {
 
 func (s *simulator) allNodesCreateBlocks() error {
 	for _, node := range s.handlers {
+		// TODO remove this when we remove all goroutines
```

Comment:
Please keep track of these todos with tasks in JIRA and reference them here.
Otherwise, these TODOs will be lost